app/deck.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any
+
+
+class Deck:
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
+
+    @staticmethod
+    def validator(data: Any) -> None:

```

Comment:
validator is not so clear name. Please choose better name, for example: 
validate_index